### DISCUSSION TITLE
What is the maximum number of concurrent users in a room?

### DISCUSSION THREAD

**Discussion by HotObsidian** [naverz/zepeto-studio-global#68] [Q&A]
What is the maximum number of concurrent users in a room? What conditions need to be met if you want to set the number of people over the limit

**Comment by naverz-mkao** [comment]
Hi HotObsidian,

Currently, the maximum number of concurrent users is 16. Unfortunately right now there is no way to go beyond 16 concurrent users in a room.

<img width="796" alt="image" src="https://user-images.githubusercontent.com/107931355/187781392-f9dccf94-9b76-427c-a00e-0a846396fc8e.png">

**Reply by HotObsidian** [reply]
It is an honor to have your reply. In fact, I am a Unity development technician of a clothing brand company. Our company wants to put a brand theme world on the zepeto platform. I would like to take this opportunity to ask a few questions:

1. We have uploaded and sold a lot of clothes products on the company's account, but we don't know how to bind the products with zepeto world products.
2. I see that many zepeto world users have the same layout and function of the interactive interface, is there a development template to provide this?
3. Our company wants to increase the maximum number of people in the room on the basis of 16 players, can we do it?

Because it is a google translation, there may be ambiguities. If there is any unclear expression, you can ask me. Looking forward to your reply.

**Reply by jhslee** [reply]
Hello @HotObsidian 👋 

1. How do I bind the clothing products to world products? 
At the moment, this clothing items cannot currently be sold as IWPs in the world, but we do have this feature in the works! You'll need to wait for the mannequin api before this can be possible. Please keep an eye on the roadmaps page:
https://github.com/orgs/naverz/projects/10
for updates on the development progress of this. Once this is available, we will have a guide out on how to get this working! 

2. Is there a development template for UI?
To encourage a unique design and UI interface for each world created, there isn't a template per say, so we only show the home, chatting, and UI as a default overlay in the world.

:small_blue_diamond:  The following are links to official sample worlds that we’re providing for UI. These will contain some common resources for you to use if you'd like to match the style of zepeto UI:
> :white_small_square: Camera: https://studio.zepeto.me/guides/screenshot
:white_small_square: Video capture: https://studio.zepeto.me/guides/video-save-share-post
:white_small_square: Quick chat: https://studio.zepeto.me/guides/quick-message

3. Can we increase the maximum number of players beyond 16?
Unfortunately 16 is a hard limit on the maximum player count at the moment, but there are plans to increase this, which I cannot quite share details at this stage. 

Hope this helps! 😁
